barretenberg/cpp/src/barretenberg/vm2/simulation/lib/merkle.cpp

```diff
@@ -23,6 +23,11 @@ FF root_from_path(const FF& leaf_value, const uint64_t leaf_index, std::span<con
     return curr_value;
 }
 
+FF silo_nullifier(const AztecAddress& contract_address, const FF& nullifier)
```

Comment:
Up to you: I think in some files we were calling these `unconstrained_silo_...` just to be clear, but no strong opinions.